ui/src/scenes/Home/components/NeedsContainer/NeedsContainer.js

```diff
@@ -4,11 +4,12 @@ import gql from 'graphql-tag';
 import _ from 'lodash';
 import { withRouter, Redirect } from 'react-router-dom';
 import { Query } from 'react-apollo';
+import { GET_NEEDS } from '@/services/queries';
+import { REALITIES_CREATE_SUBSCRIPTION, REALITIES_DELETE_SUBSCRIPTION, REALITIES_UPDATE_SUBSCRIPTION } from '@/services/subscriptions';
```

Comment:
I'd say break this up into multiple lines.

```
import {
  REALITIES_CREATE_SUBSCRIPTION,
  REALITIES_DELETE_SUBSCRIPTION,
  REALITIES_UPDATE_SUBSCRIPTION,
} from '@/services/subscriptions';
```

Reply:
Fixed